.github/workflows/industrial_ci_action.yml

```diff
@@ -9,48 +9,41 @@ env:
   CMAKE_ARGS: "-DCATKIN_ENABLE_CLANG_TIDY=true"
 
 jobs:
-  build_test_main:
-    name: "Build + Test with Main Repo of melodic (http://packages.ros.org/ros/ubuntu)"
-    runs-on: ubuntu-18.04
-    steps:
-      - uses: actions/checkout@v1
-      - uses: 'ros-industrial/industrial_ci@master'
+  builds:
+    strategy:
+      matrix:
         env:
-          ROS_REPO: main
-          ROS_DISTRO: melodic
-    continue-on-error: true
-
-  build_test_testing:
-    name: "Build + Test with Testing Repo of melodic (http://packages.ros.org/ros-testing/ubuntu)"
-    runs-on: ubuntu-18.04
+          - {ROS_DISTRO: melodic, ROS_REPO: testing}
+          - {ROS_DISTRO: melodic, ROS_REPO: main}
```

Comment:
How exactly are we going to handle cases, where `testing` runs but `main` doesn't?
See https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idcontinue-on-error

Reply:
Added `continue-on-error` which should resolve the issue. -> Done.